app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -57,6 +86,13 @@ public Product deleteProduct(Long id) {
         return product;
     }
 
+    /**
+     * 식별 값으로 상품을 조회하고, 조회 된 상품을 리턴합니다.
+     * @param id 식별 값
+     * @return 조회 된 상품
+     * @throws ProductNotFoundException
+     *         식별 값으로 상품이 조회되지 않을 때 예외를 던집니다.
+     */
```

Comment:
`@throws`에서 "예외를 던진다"는 말은 이미 `@throws`에 들어 있어요. 그래서 굳이 "예외를 던집니다"라고 써주지 않아도 좋아요.

```suggestion
    /**
     * 식별 값으로 상품을 조회하고, 조회 된 상품을 리턴합니다.
     * @param id 식별 값
     * @return 조회 된 상품
     * @throws ProductNotFoundException 상품을 찾지 못했을 경우
     */
```